charts/fission-all/templates/fluentbit/fluentbit.yaml

```diff
@@ -173,7 +173,6 @@ spec:
             - name: fluentbit-config
               mountPath: /fluent-bit/etc/
               readOnly: true
-      serviceAccountName: fission-svc
```

Comment:
This pod containers logger container, requires a pod get/list/watch permissions.

I see function log test failed.

```
2022-10-04T06:55:34.546352417Z stderr F E1004 06:55:34.546231       1 reflector.go:140] pkg/mod/k8s.io/client-go@v0.25.2/tools/cache/reflector.go:169: Failed to watch *v1.Pod: failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:fission:default" cannot list resource "pods" in API group "" at the cluster scope
```